queries.sql

```diff
@@ -27,3 +27,86 @@ SELECT * FROM ANIMALS WHERE NAME != 'Gabumon';
 */
 SELECT * FROM ANIMALS WHERE WEIGHT_KG BETWEEN 10.4 AND 17.3;
 
+
+/* 
+
+  transactions 
+
+*/
+
+/* Update the animals table by setting the species column to unspecified. Verify that change was made. Then rolled back the change and verified that the species columns went back to the state before the transaction. */
+
+BEGIN;
+UPDATE ANIMALS SET SPECIES = 'unspecified';
+SELECT * FROM ANIMALS;
+ROLLBACK;
```

Comment:
* As stated in the [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction.), Kindly verify that the species columns went back to the state before the transaction after rolling back the change. HINT 💭 running these queries  again  (`SELECT species from animals;`) verifies that the species column went back to original state

Reply:
I had added the checking for all the transactions in my screenshots queries with a "select *" query. Adding them to the queries.sql as well then.